modules/nw-ingress-controller-configuration-gcp-global-access.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-gcp-global-access_{context}"]
+= Configuring Global Access parameters on GCP
+As a cluster administrator with an internal load balancer on GCP, the global access option enables clients from any region within the load balancer's VPC network to reach the workloads running on your cluster.
```

Comment:
The ISG generally advises against using apostrophes. I think this looks good and makes sense, so I'll leave any changes on this one up to you.